src/main/java/com/qaprosoft/carina/demo/gui/pages/PhoneModelPage.java

```diff
@@ -90,8 +91,14 @@ public void sortByNewFirst() {
     public boolean areCommentsSortedByDate() {
         List<Date> dates = new ArrayList<>();
         List<Integer> hours = new ArrayList<>();
+        List<Integer> minutes = new ArrayList<>();
+        List<Integer> seconds = new ArrayList<>();
         dateComments.stream().forEach(t -> {
-            if (t.getText().contains("ago")) {
+            if (t.getText().contains("seconds ago") || t.getText().contains("second ago")) {
+                seconds.add(Integer.parseInt(t.getText().replaceAll("\\D+", "")));
+            } else if (t.getText().contains("minutes ago") || t.getText().contains("minute ago")) {
+                minutes.add(Integer.parseInt(t.getText().replaceAll("\\D+", "")));
+            } else if (t.getText().contains("hours ago") || t.getText().contains("hour ago")) {
```

Comment:
if you use contains remove condition with letter s
or replace all contains to equals